src/FSAL/FSAL_CORTXFS/conf/nfs_setup.sh

```diff
@@ -339,8 +363,8 @@ Options:
   -p Prompt
   -q To perform Setup on Provisioner VM
   -d To create FS
-  -r pNFS Enabled
-  -D pNFS Data-Server IP Addr 
+  -r pNFS Role {MDS|DS|BOTH}.If pNFS should be disabled do not specify -r option
+  -D pNFS Data-Server IP Addr for MDS and BOTH options {IPADDR1,IPADDR2...IPADDRN} 
```

Comment:
In future, we should support multiple IP addresses per DS. This format can be updated to use an array of IPv4 address arrays.
e.g. 
 {IPADDR1,IPADDR2...IPADDRN} ,  {IPADDR1,IPADDR2...IPADDRN} , .....,  {IPADDR1,IPADDR2...IPADDRN}

Reply:
@sachinpunadikar , i remember discussing this in scrum that we will support only single IP address per data server.If that is not the case i will file a ToDo ticket for this work as well

Reply:
I agree with Pratyush .. a DS may have multiple IP addresses. So this need to be considered.

Reply:
Have created EOS-13899 for this and linked it to pNFS epic